tests/rbd/rbd_clone_delete_parent_snapshot.py

```diff
@@ -0,0 +1,99 @@
+"""Module to verify for parent snapshot deletion after flattening the clone.
+
+    Pre-requisites :
+    We need atleast one client node with ceph-common package,
+    conf and keyring files
+
+    Test cases covered -
+    1) CEPH-83573650 - Create image with Deep Flattening enabled,take snap,protect,clone,snap,
+    flatten the clone, unprotect the parent snap, delete the parent snap.
+
+    Test Case Flow -
+    1. Create a Rep pool and an Image
+    2. Create Snapshot and protect, and clones from it.
+    3. Create a snapshot of clone.
+    4. Flatten the clone.
+    5. Unprotect and delete the parent snapshot
+    6. Repeat above steps on EC pool
+    """
```

Comment:
Could you please correct the indentation on this line

Reply:
Done.